recipe_scrapers/_utils.py

```diff
@@ -229,14 +248,10 @@ def change_keys(obj, convert):
     (https://web.archive.org/web/20201022163147/https://stackoverflow.com/questions/11700705/python-recursively-replace
```

Comment:
Let's add a 'note: with modifications applied' to the credit, since this code has been adapted.